README.md

```diff
@@ -77,8 +77,9 @@ This project is aimed at helping you better understand how the type system works
 
 ### Projects / Solutions
 
-- [Type Gymnastics](https://github.com/g-plane/type-gymnastics)
+- [🎥 Video Explanations and Solutions for every challenge! 🔥](https://www.youtube.com/playlist?list=PLOlZuxYbPik180vcJfsAM6xHYLVxrEgHC)
```

Comment:
```suggestion
- [🎥 Video Explanations and Solutions for every challenge!](https://www.youtube.com/playlist?list=PLOlZuxYbPik180vcJfsAM6xHYLVxrEgHC)
```

While I agree you did a great job on this, having too attractive emojis can be a bit unfair to others. I am ok with keeping the 🎥 indicating they are videos

Reply:
I didn't think about it that way, thanks!